.kiro/specs/auterity-ai-platform-unified/templates/preview-api-specification.py

```
@@ -0,0 +1,748 @@
+            
+            # Process based on node type
+            if request.node_type == NodeType.DATA_SOURCE:
+                result = await self._process_data_source(request, sandbox_id)
+            elif request.node_type == NodeType.TRANSFORM:
+                result = await self._process_transform(request, sandbox_id)
+            elif request.node_type == NodeType.VISUALIZATION:
+                result = await self._process_visualization(request, sandbox_id)
+            elif request.node_type == NodeType.CONTENT:
+                result = await self._process_content(request, sandbox_id)
+            else:
+                raise ValueError(f"Unsupported node type: {request.node_type}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)